cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -333,3 +427,79 @@ def test_analysis_profile_description_validation(analysis_profile_collection):
     )
     view.flash.assert_message("Description can't be blank")
     view.cancel.click()
+
+
+@test_requirements.rbac
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream('5.9')
+@pytest.mark.parametrize(
+    'product_features',
+    [  # Navigation for product features trees
+        # product_features tree for Managing Quotas for 'My Company' tenant
+        PRODUCT_FEATURES[0],
```

Comment:
here is where the extra layer of lists are in the above data structure. Why isn't PRODUCT_FEATURES, a list, passed directly to parametrize here? Why the extra layer and specific index references here? The comments here should be in the definition above, where there is no context as to what is being defined.

Reply:
While updating role with ```product_feature``` tree. we need to pass product_feature as a list of tupple with true/fasle arguments to check or uncheck the node.